duckduckgo/Android
# Feature/4.1 widget

Asana Issue URL: https://app.asana.com/0/414730916066338/532267986399037

## Description
Add simple widget which launches straight into view ready for searching. 


## Steps to Test this PR:
1. Install the app
1. Add widget to the home screen
1. Test that the widget can be slightly adjusted horizontally
1. Click on the widget; ensure it launches the app with the keyboard shown and the search field focussed
